pkgs/top-level/python-packages.nix

```diff
@@ -6640,7 +6646,7 @@ self: super: with self; {
   maya = callPackage ../development/python-modules/maya { };
 
   mayavi = pkgs.libsForQt5.callPackage ../development/python-modules/mayavi {
-    inherit buildPythonPackage pythonOlder;
+    inherit buildPythonPackage pythonOlder pythonAtLeast;
```

Comment:
We should get this from python

Reply:
moving this down to the inherit with (self) should fix this right?